apps/admin/frontend/test/render_in_app_context.tsx

```diff
@@ -89,7 +87,13 @@ export function renderRootElement(
     apiClient = createMockApiClient(),
     // TODO: Determine why tests fail when using createQueryClient and, by extension,
     // QUERY_CLIENT_DEFAULT_OPTIONS
-    queryClient = new QueryClient(),
+    queryClient = new QueryClient({
+      defaultOptions: {
+        queries: {
+          staleTime: Infinity,
+        },
+      },
+    }),
```

Comment:
🙈 I discovered this discrepancy between the query client in tests and the query client in source code a while back, hence the TODO

To minimize the differences between the test query client and the real query client without having to get to the bottom of the test failures when using the default options, might be nice to still use the default options but with explicit overrides, e.g.

```ts
queryClient = new QueryClient({
  defaultOptions: {
    ...QUERY_CLIENT_DEFAULT_OPTIONS,
    queries: {
      ...(QUERY_CLIENT_DEFAULT_OPTIONS.queries ?? {}),
      useErrorBoundary: false,
    },
  },
}),
```

Doing a quick manual test, I think it's just `queries.useErrorBoundary: true` that's now resulting in test failures